xml/containers_basics.xml

```diff
@@ -1,78 +1,211 @@
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        It is possible to control network interfaces and apply resources inside
+        containers through
+        <xref linkend="gloss-vt-lxc-cgroups"/>
+        .
+      </para>
+    </listitem>
+  </itemizedlist>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Limitations of Containers</title>
+    <listitem>
+      <para>
+        Containers run on the host system's kernel, and they cannot use
+        different kernels or different kernel versions.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Only Linux-based applications can be containerized.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers are not secure, and the overall security depends on the host
+        system. Containerized applications can be secured through &aa; or
+        &selnx; profile. Securing containers is harder than securing virtual
+        machines due to the larger attack surface.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <sect1 xml:id="sec-concepts-workflow">
+    <title>Key Concepts and Brief Introduction to &podman;</title>
+    <para>
+      Although &deng; is a popular choice for working with images and containers,
+      &podman; provides a &docker; drop-in replacement that offers several
+      advantages. While
+      <xref linkend="sec-build-tools-podman" />
+      provides an
+      in-depth information on &podman;, this chapter offers a quick introduction
+      to key concepts and a basic procedure of creating a container image and
+      using it to run a container.
+    </para>
+    <para>
+      The basic &podman; workflow is as follows:
+    </para>
+    <informalfigure>
+      <mediaobject>
+        <imageobject role="fo">
+          <imagedata fileref="container-workflow.png" width="80%" format="PNG"/>
+        </imageobject>
+        <imageobject role="html">
+          <imagedata fileref="container-workflow.png" width="100%" format="PNG"/>
+        </imageobject>
+      </mediaobject>
+    </informalfigure>
+
+    <para>
+      Running a container on a local machine or cloud service, usually involves the following steps:
+    </para>
+    <orderedlist>
+      <listitem>
+        <para>
+          Fetch a base image by pulling it from a registry to your local machine
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Create a Dockerfile and use it to build a custom image on top of the base image
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Use the created image to run one or multiple containers
+        </para>
+      </listitem>
+    </orderedlist>
+    <para>
+      To run a container, you need an image. An image includes all the
+      dependencies needed to run the application. For example, the SLE base
+      image contains the SLE distribution with a minimal package selection.
```

Comment:
Is "SLE" the name of an image here, or part of the name? If so I think it should be formatted in some way to indicate that. If not, shouldn't we use the &sle; entity?